sirepo/template/ml.py

```diff
@@ -808,6 +853,23 @@ def _read_file_with_history(run_dir, filename, report=None):
     return res
 
 
+def _remote_data_bytes_loaded(filename):
+    import os
```

Comment:
This can be up top

Reply:
[3000344](https://github.com/radiasoft/sirepo/pull/4671/commits/3000344484fe9bb5b46259d9515b146e2c99a068)